docs/misc/changelog.rst

```diff
@@ -13,12 +13,15 @@ Breaking Changes:
 - Removed shared layers in ``mlp_extractor`` (@AlexPasqua)
 - Refactored ``StackedObservations`` (it now handles dict obs, ``StackedDictObservations`` was removed)
 - You must now explicitely pass a ``features_extractor`` parameter when calling ``extract_features()``
+- Dropped offline sampling for ``HerReplayBuffer``
```

Comment:
new her replay buffer is only not backward compatible, no?

Reply:
Indeed. I don't know how much effort it would take to make it backward compatible.